libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/RustMatrixClient.kt

```diff
@@ -534,6 +534,10 @@ class RustMatrixClient(
         return client.availableSlidingSyncVersions().contains(SlidingSyncVersion.Native)
     }
 
+    override suspend fun isSlidingSyncProxySupported(): Boolean {
+        return client.availableSlidingSyncVersions().any { it is SlidingSyncVersion.Proxy }
```

Comment:
Can't this fail?

Reply:
Apparently not, if the SDK can't load the data (network error, one of the config files not existing...) it just returns an empty list.